app/code/Magento/Analytics/Model/Cryptographer.php

```diff
@@ -135,6 +135,6 @@ private function validateCipherMethod($cipherMethod)
         );
         $cipherMethod = strtolower($cipherMethod);
 
-        return (false !== array_search($cipherMethod, $methods));
+        return in_array($cipherMethod, $methods);
```

Comment:
I would like to replace both array_search & in_array to array_flip + isset.
Reason: https://github.com/magento/magento-coding-standard/issues/143

@carusogabriel @osrecio what do you think about it?

Reply:
Followin the thread I think is better to usee `array_flip` + `isset`.

If @carusogabriel agree with us, he can perform the changes and for me will be ready to be merged :)

Reply:
@osrecio I'll do it :)